module-3/README.md

```diff
@@ -66,7 +76,7 @@ import logging
 import ask_sdk_core.utils as ask_utils
 import os
 from ask_sdk_s3.adapter import S3Adapter
-s3_adapter = S3Adapter(bucket_name=os.environ["S3_PERSISTENCE_BUCKET"])
+s3_adapter = S3Adapter(bucket_name="NAME_OF_YOUR_BUCKET_HERE")
```

Comment:
Why are we removing `os.environ` from here? That's how an Alexa Hosted skill references an S3 bucket created by Alexa Hosted.

Reply:
Using os.environ["my_bucket_name"] throws errors during testing. The log is not very descriptive but this is what it looks like.
![os_environ_error](https://user-images.githubusercontent.com/3944228/91786533-ae5e0e00-ebbc-11ea-86ab-a650a187df07.PNG)